src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -263,6 +264,27 @@ class SourcesTree extends Component<Props, State> {
     showMenu(event, menuOptions);
   };
 
+  onExpand = (item, expandedState) => {
```

Comment:
General comment on this file's changes, love the clean-up. It's clearer what you're doing and things are organized more logically.